src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -36,9 +40,9 @@
 
 import java.io.IOException;
 import java.nio.file.Files;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
+import static io.mosip.mimoto.service.IssuersServiceTest.getIssuerDTO;
```

Comment:
Why are we referring to IssuersServiceTest for DTO?
create DTO required for controller.

Reply:
Refactored